apps/ledger-live-desktop/src/renderer/modals/StorylyDebugger/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Trans } from "react-i18next";
+import { StorylyWrapper, storyInstancesIDsMap } from "~/renderer/components/Storyly";
+import { Box, Text } from "@ledgerhq/react-ui";
+import Alert from "~/renderer/components/Alert";
+import Modal, { ModalBody } from "~/renderer/components/Modal";
+import { ScrollArea } from "~/renderer/components/Onboarding/ScrollArea";
+
+const StorylyDebugger = ({ name }: { name: string }) => {
+  return (
+    <Modal
+      name={name}
+      centered
+      render={() => (
+        <ModalBody
+          onBack={undefined}
```

Comment:
shouldn't there be an `onClose` here?
From what I see elsewhere you can do it like this:
```diff
return (
    <Modal
      name={name}
      centered
+     render={({ onClose }) => (
        <ModalBody
+         onClose={onClose}
        />
       )}
    />
);